src/presentation/styles/sizes.pcss

```diff
@@ -17,4 +17,7 @@
   --layout-content-width: 700px;
 
   --layout-header-height: 52px;
+
+  --header-tab-max-width: 138px;
+  --header-tab-height: 32px;
```

Comment:
--header-tab-height: 32px;

do we really need this?

Reply:
removed